nixos/modules/services/video/mirakurun.nix

```diff
@@ -0,0 +1,155 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mirakurun;
+  mirakurun = pkgs.mirakurun;
+  username = config.users.users.mirakurun.name;
+  groupname = config.users.users.mirakurun.group;
+  settingsFmt = pkgs.formats.yaml {};
+in
+  {
+    options = {
+      services.mirakurun = {
+        enable = mkEnableOption mirakurun.meta.description;
+
+        logLevel = mkOption {
+          type = types.ints.between (-1) 3;
+          default = 2;
+          description = "Mirakurun log level.";
```

Comment:
Could this option be left out? I'm not sure if it's too useful to expose explicitly, especially since it can be set with `serverSettings.logLevel` too

Reply:
Fixed